tedana/tests/test_integration.py

```diff
@@ -277,6 +278,48 @@ def test_integration_five_echo(skip_integration):
     check_integration_outputs(fn, out_dir)
 
 
+def test_integration_robustica_five_echo(skip_integration):
+    """Integration test of the full tedana workflow with robustica using five-echo test data."""
+
+    if skip_integration:
+        pytest.skip("Skipping five-echo integration test")
+
+    test_data_path, osf_id = data_for_testing_info("five-echo")
+    out_dir = os.path.abspath(os.path.join(test_data_path, "../../outputs/five-echo"))
+    # out_dir_manual = f"{out_dir}-manual"
+
+    if os.path.exists(out_dir):
+        shutil.rmtree(out_dir)
+
+    # if os.path.exists(out_dir_manual):
+    #     shutil.rmtree(out_dir_manual)
```

Comment:
Preferable to not have novel commented-out code added.

Reply:
Indeed this code is uncommented in the four-echo and three-echo robustica test functions. If there's an intention to have some kind of manual override of data location, it should be consistent. Or if this code does nothing in any of the functions, it should not be included in the PR.

Reply:
The issue is fixed now. I only removed it from my part.